test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -362,7 +362,60 @@ static public void testWithFieldOnNonExistentClass() throws Throwable {
 			Assert.fail("should throw error. Class does not exist");
 		} catch (NoClassDefFoundError e) {}
 	}
-	
+
+	/*
+	 * Test ifacmp on value class
+	 * 
+	 * class TestIfacmpOnValueClass {
+	 *  
+	 * }
+	 */
+	@Test(priority=1)
```

Comment:
Where you able to run this test? I think this has to be priority 2 because testCreatePoint2D is priority 1

Reply:
yes, because I actually create a new value type here, but I will test it again and tell you the result later

Reply:
I have retested it and my test is able to run

Reply:
`point2DClass = ValueTypeGenerator.generateValueClass("Point2DInAcmp", fields);` this overwrites a shared global that is used by other test cases. Its not a problem now, but may be in the future.

You can simply change the priority to 2, and use the global `makePoint2D` instead of re-assigning here

Reply:
This has been resolved.